src/Graphics/Model.elm

```diff
@@ -0,0 +1,97 @@
+module Graphics.Model where
+
+import Color (Color, Gradient)
+import Maybe (Maybe)
+import Transform2D (Transform2D)
+
+type alias Form =
+    { theta : Float
+    , scale : Float
+    , x : Float
+    , y : Float
+    , alpha : Float
+    , form : BasicForm
+    }
+
+type BasicForm
+    = FPath LineStyle Path
+    | FShape ShapeStyle Shape
+    | FImage Int Int (Int,Int) String
+    | FElement Element
+    | FGroup Transform2D (List Form)
+
+type alias Path = List (Float,Float)
+
+type alias Shape = List (Float,Float)
+
+{-| The shape of the ends of a line. -}
+type LineCap = Flat | Round | Padded
```

Comment:
In previous versions, these constructors were exposed. Now there is no way to construct something of type `LineCap`. I worry that we are doing bad changes as a stopgap for the real answer here.

Another way to get the same benefit in code size is to write a crash function in `Native.Signal` and then we don't have to do any of this. Or move `Debug.crash` into its own thing separate from `Debug.trace`